tests/test_configmanager.py

```diff
@@ -159,33 +161,28 @@ def test_parse(self):
         self.assertIn('he-ipv6', self.configmanager.virtual_interfaces)
 
     def test_parse_merging(self):
-        self.configmanager.parse(extra_config=[os.path.join(self.workdir.name, "newfile_merging.yaml")])
-        self.assertIn('eth0', self.configmanager.ethernets)
-        self.assertIn('dhcp4', self.configmanager.ethernets['eth0'])
-        self.assertEquals(True, self.configmanager.ethernets['eth0'].get('dhcp6'))
-        self.assertEquals(True, self.configmanager.ethernets['ethbr1'].get('dhcp4'))
+        state = self.configmanager.parse(extra_config=[os.path.join(self.workdir.name, "newfile_merging.yaml")])
+        self.assertIn('eth0',   state.ethernets)
```

Comment:
nitpick (non-blocking): whitespace offset.

Also, we're droping some tests of the values, which are not exported via libnetplan's API, so that's fine.

Reply:
Here and elsewhere, I'm not addressing the value dropped, since as you noted it's not exactly easy to fetch them. Whitespace fixed, though.